pages/dkp/kaptain/2.0.0/install/index.md

```diff
@@ -23,11 +23,11 @@ The infrastructure you select determines the specific requirements for a success
 
 ## Before you begin
 
-Kaptain is a DKP Catalog application. To use it, simply add it to your repository and deploy it on selected Workspaces. No downloads are necessary.
+Kaptain is a DKP Catalog application. To use it, add it to your repository and then deploy it on selected Workspaces. No downloads are necessary.
 
 You can deploy Kaptain to single and multi-clusters environments. The difference between these setups is the following: In a single-cluster environment (with an Essential license), you run on one Management cluster only. In a multi-cluster environment (with an Enterprise license), you run Kaptain on one or several Managed and Attached clusters.
 
-Kaptain is deployed on a per-Workspace basis. If you are running a multi-cluster environment with several Workspaces, Kaptain is deployed to every cluster within a Workspace.
+You can deploy Kaptain on a per-Workspace basis. If you are running a multi-cluster environment with several Workspaces, Kaptain is deployed to every cluster within the selected Workspace. Workspaces can contain both Managed and Attached clusters, as a result, Kaptain deploys to both.
```

Comment:
@chessahoekstra just want to be 100% that this is correct :)

Reply:
```suggestion
You can deploy Kaptain on a per-workspace basis. If you are running a multi-cluster environment, Kaptain is deployed to every cluster within the selected workspace.
```
this sounds good +concise to me - not sure if we need to specify that note about attached/managed clusters (though I did verify it it correct). but ultimately up to you on final touches!

Reply:
@gvasquezvargas